README.md

```diff
@@ -83,15 +83,16 @@ ones controlled by your test.
 
 We provide a top-level method called `td.replace()` that operates in two
 different modes: CommonJS module replacement and object-property replacement.
-Both modes will, by default, perform a deep clone the real dependency which
-replaces all functions it encounters with fake test double functions that can be
-configured by your test to either stub responses or assert invocations.
+Both modes will, by default, perform a deep clone of the real dependency,
```

Comment:
>Both modes will, by default, perform a deep clone the real dependency

This sentence seemed like it was missing a preposition before "the real dependency". "of" felt right here, but I could be wrong!